tests/python/rest_api/test_webhooks_sender.py

```diff
@@ -385,7 +385,7 @@ def test_webhook_create_and_delete_task(self, organizations):
                 delete_payload["task"],
                 task,
                 ignore_order=True,
-                exclude_paths=["root['updated_date']"],
+                exclude_paths=["root['updated_date']", "root['jobs']"],
```

Comment:
Why should we exclude it here?

Reply:
There is no `Data` object after the task is removed, so this field cannot be computed.